src/client/containers/SessionTokenGuard.test.tsx

```diff
@@ -13,48 +12,51 @@ jest.mock('../apolloClient', () => ({
   },
 }))
 
+jest.mock('react-router-dom', () => {
+  return {
+    Redirect: jest.fn(({ to }) => `Redirected to ${to}`),
+  }
+})
+
 it('redirects when there is no session', () => {
-  const wrapper = mount(
-    <StaticRouter location={'/se/new-member/blah'} context={{}}>
-      <MockedProvider>
-        <Provider
-          initialState={{
-            storage: { session: createSession(new MockStorage()) },
-          }}
-        >
-          <SessionTokenGuard>
-            <div />
-          </SessionTokenGuard>
-        </Provider>
-      </MockedProvider>
-    </StaticRouter>,
+  const { getByText } = renderComponent(
+    <MockedProvider>
+      <Provider
+        initialState={{
+          storage: { session: createSession(new MockStorage()) },
+        }}
+      >
+        <SessionTokenGuard>
+          <div />
+        </SessionTokenGuard>
+      </Provider>
+    </MockedProvider>,
+    { wrapperProps: { location: { pathname: '/se/new-member/blah' } } },
   )
 
-  expect(wrapper.find(Redirect).prop('to')).toBe('/se/new-member')
+  expect(getByText('Redirected to /se/new-member')).toBeInTheDocument()
```

Comment:
This comes from the `react-router-dom` mock that I setup earlier in the module.

The mock just makes it so that some text is printed when a `<Redirect />` component is rendered. I think it would otherwise be hard to test that such a component is rendered when expected.

I agree that this is kind of testing implementation details but it does cover the current use. Perhaps it's okey for this PR and then we can continue improving it!